src/dev/i18n/tasks/check_default_messages_in_files.ts

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
A little bit weird why we use the large license header in a new file.

Reply:
code is copied from a file that has other license.